src/content/guides/farmlands-portal.mdx

```diff
@@ -195,55 +197,51 @@ On the Settings page, you can change your business information. Please note this
-
-If the entire order has been cancelled, release the hold on the customer’s funds via the Portal. There is no requirement to send an invoice to Farmlands.
-
-**What do we do if we need to amend the value of the order prior to sending the invoice to Farmlands?**
-
-If the revised sale amount is:
-- less than the original authorisation amount, submit the amended invoice with the original authorisation short code.
-- greater than the original authorisation sales amount, release the hold on the customer’s funds via the Portal. Once completed, generate a new authorisation request for the revised sales amount.
-
-**How long are funds held for on a Customer’s Card?**
-
-The value of the authorisation will be held on the customer’s Card for up to 31 days. It is important Invoices, with a valid authorisation short code, are submitted to Farmlands as soon as possible so these can be matched against the funds held.
-
-Invoices received after 31 days and up to 3 months will be processed in good faith however, if the associated Farmlands Account has been closed, you will be advised to contact the customer for another form of payment.
-
-If you are concerned, **re-authorise the transaction especially prior to fulfilling a back order if it is over 31 days since originally authorised.**
-
-**We received a message on the Portal stating the “Barcode is invalid.” What does this mean?**
-
-The Card number being entered is either not a valid number or the Card being presented is no longer active on the Farmlands system. Check the correct Card number has been entered or request another method of payment.
-
-**How do I manage who has access to the Authorisation Portal?**
-
-User management and access is at the discretion of the Card Partner.
-
-**We have a regular customer. Do we still need to obtain an authorisation?**
-
-All sales where a Farmlands Card has been provided as payment require an authorisation prior to submitting an invoice to Farmlands.
-
-If you hold your customer’s Farmlands’ Card number on file and have their permission to charge this on a recurring basis, please select the 'Cardholder Not Present' option at the time of sale.
-
-**What if the barcode will not scan or is obscured?**
-
-The 9-digit Card number can be entered manually. This is printed on the back below the barcode and on the front of the Card under the Cardholder's name.
-
-If neither is visible, please request another method of payment. Advise the Cardholder to contact Farmlands to request a new Card.
-
-**The authorisation request has prompted us to “Check ID.” The customer is not the actual Cardholder but one of the workers of the Cardholder. Can I accept the Card?**
-
-No, unfortunately you must only accept the Card if the true owner of the Card is making the purchase.
+<FAQ client:visible title="What do we do if a customer returns one or more items, and the invoice has already been sent to Farmlands?">
```

Comment:
I'm not a fan of this, I think logically there needs to be a wrapping component for all of them.

This is how you might track state in the future to have one open at a time.
Or in your case, insert a line in-between disclosures.

Looking at what FAQ does, it's adding a single style and then passing the slot and props through.
I suspect there's concern about adding this style to the disclosure as we may want to re-use disclosure.
I think if that's the case - get rid of Dislocsure.vue and call it FaqDisclosure.vue, you don't need that extra layer of abstraction.